python/tests/server/test_http_input.py

```diff
@@ -191,7 +192,7 @@ class Input(BaseModel):
         text: str
 
     class Predictor(BasePredictor):
-        def predict(self, input: Input):
+        def predict(self, input: Input) -> Any:
```

Comment:
In a real predictor, I suppose this would actually be `str`, but in this case the test isn't testing the output types so it doesn't matter.

Reply:
I've updated both tests to use `str` because it's a slightly better representation of real world usage, and it avoids needing the import in both tests.